openc3/lib/openc3/system/system.rb

```diff
@@ -72,6 +75,14 @@ def self.limits_set=(value)
       @@limits_set = value.to_s.intern
     end
 
+    def self.add_post_instance_callback(callback)
+      if @@instance
+        callback.call()
+      else
+        @@post_instance_callbacks << callback
+      end
+    end
```

Comment:
What are the plans for this?

Reply:
I use it as a lifecycle hook to be able to define dynamic packets, after the regular system is fully created.